gengokit/middlewares/templates/endpoints.go

```diff
@@ -5,9 +5,20 @@ package middlewares
 
 import (
 	"github.com/go-kit/kit/endpoint"
+	svc "{{.ImportPath -}} /generated"
 )
 
-func InjectEndpointMiddlewares(in endpoint.Endpoint) endpoint.Endpoint {
+// WrapEndpoint will be called individually for all endpoints defined in
+// the service. Implement this with the middlewares you want applied to
+// every endpoint.
+func WrapEndpoint(in endpoint.Endpoint) endpoint.Endpoint {
```

Comment:
Just throwing in here, upon first review of this PR reading the names of the functions as they are alongside their comments left me with no confusion as to their purpose.

I would even cast a vote to potentially just leave them.